book/_config.yml

```
@@ -0,0 +1,44 @@
+# Book settings
+title: Template documentation jupyter book
+author: University of Leeds Research Computing Team
+logo: ./assets/img/logo/logo.png
+email: ask-rc@leeds.ac.uk
+copyright: "2020"
+
+# execute settings
+execute:
+  execute_notebooks: auto
+
+# HTML-specific settings
+html:
+  favicon                   : "./assets/img/logo/favicon-32x32.png"  # A path to a favicon image
+  navbar_number_sections    : False  # Add a number to each section in your left navbar
+  google_analytics_id       : ""  # A GA id that can be used to track book views.
+  home_page_in_navbar       : true  # Whether to include your home page in the left Navigation Bar
+  use_repository_button: true
+  use_issues_button: true
+  use_edit_page_button: true
+  baseurl                   : "https://arctraining.github.io/template-jb-docs"  # The base URL where your book will be hosted. Used for creating image previews and social links. e.g.: https://mypage.com/mybook/
+  navbar_footer_text        :
+    'Visit the <a href="https://arcleeds.github.io/">Research Computing Home Site</a>
+    <div>
+    This book is powered by <a href="https://jupyterbook.org">Jupyter Book</a>
+    </div>'  # Will be displayed underneath the left navbar.
+
+
+# Launch button settings
+launch_buttons:
+  notebook_interface        : "classic"  # The interface interactive links will activate ["classic", "jupyterlab"]
+  binderhub_url             : "https://mybinder.org"  # The URL of the BinderHub (e.g., https://mybinder.org)
+  jupyterhub_url            : ""  # The URL of the JupyterHub (e.g., https://datahub.berkeley.edu)
+  thebelab                  : false  # Add a thebelab button to pages (requires the repository to run on Binder)
+
+repository:
+  url                       : https://github.com/ARCTraining/template-jb-docs  # The URL to your book's repository
+  path_to_book              : "book/"  # A path to your book's folder, relative to the repository root.
+  branch                    : gh-pages  # Which branch of the repository should be used when creating links
```

Not sure here. I am used to put `main`.

```suggestion
  branch                    : main  # Which branch of the repository should be used when creating links
```